### ISSUE TITLE
segmentation fault (core dumped)

### ISSUE THREAD

**Issue by birdflyto** [NVIDIA/vid2vid#140]
i have tested on several cuda,cudnn and pytorch version ,the latest vesion is pytorch1.0.1 cuda9.0 cudnn7.1.2,but all the version met the same error(segmentation fault(core dumped)). i have no idea to solve the problem.
Many thanks!!!

 python train.py --name flame --dataroot ./datasets/ --input_nc 3 --ngf 128 --max_frames_per_gpu 4 --n_frames_total 15 --niter 20 --niter_decay 20 --save_epoch_freq 10
------------ Options -------------
TTUR: False
add_face_disc: False
basic_point_only: False
batchSize: 1
beta1: 0.5
checkpoints_dir: ./checkpoints
continue_train: False
dataroot: ./datasets/
dataset_mode: temporal
debug: False
densepose_only: False
display_freq: 100
display_id: 0
display_winsize: 512
feat_num: 3
fg: False
fg_labels: [26]
fineSize: 512
fp16: False
gan_mode: ls
gpu_ids: [0]
input_nc: 3
isTrain: True
label_feat: False
label_nc: 0
lambda_F: 10.0
lambda_T: 10.0
lambda_feat: 10.0
loadSize: 512
load_features: False
load_pretrain:
local_rank: 0
lr: 0.0002
max_dataset_size: inf
max_frames_backpropagate: 1
max_frames_per_gpu: 4
max_t_step: 1
model: vid2vid
nThreads: 2
n_blocks: 9
n_blocks_local: 3
n_downsample_E: 3
n_downsample_G: 3
n_frames_D: 3
n_frames_G: 3
n_frames_total: 15
n_gpus_gen: 1
n_layers_D: 3
n_local_enhancers: 1
n_scales_spatial: 1
n_scales_temporal: 2
name: flame
ndf: 64
nef: 32
netE: simple
netG: composite
ngf: 128
niter: 20
niter_decay: 20
niter_fix_global: 0
niter_step: 5
no_canny_edge: False
no_dist_map: False
no_first_img: False
no_flip: False
no_flow: False
no_ganFeat: False
no_html: False
no_vgg: False
norm: batch
num_D: 2
openpose_only: False
output_nc: 3
phase: train
pool_size: 1
print_freq: 100
random_drop_prob: 0.05
random_scale_points: False
remove_face_labels: False
resize_or_crop: scaleWidth
save_epoch_freq: 10
save_latest_freq: 1000
serial_batches: False
sparse_D: False
tf_log: False
use_instance: False
use_single_G: False
which_epoch: latest
-------------- End ----------------
CustomDatasetDataLoader
dataset [TemporalDataset] was created
#training videos = 1
vid2vid
---------- Networks initialized -------------
-----------------------------------------------
---------- Networks initialized -------------
-----------------------------------------------
create web directory ./checkpoints/flame/web...
Segmentation fault (core dumped)

**Comment by Ha0Tang** [comment]
+1

**Comment by birdflyto** [comment]
font{
        line-height: 1.6;
    }
    ul,ol{
        padding-left: 20px;
        list-style-position: inside;
    }


    
    是环境配置的问题，可以参考我的环境配置
    
    
     
    
        
                    
            
                
                    
                            
                                
                            
                            
                                1521361730
                            
                    
                        
                            
                                    1521361730@qq.com
                            
                        
                
            
        
        
    
    
        签名由
        网易邮箱大师
        定制
    
 

    On 3/18/2020 21:31，Hao Tang<notifications@github.com> wrote： 



+1

—You are receiving this because you authored the thread.Reply to this email directly, view it on GitHub, or unsubscribe.

**Comment by birdflyto** [comment]
i've solved it,it's the misbatch of the enviornment.you can refer to my enviornment.yaml``
name: pytorch1.0
channels:
  - https://mirrors.tuna.tsinghua.edu.cn/anaconda/pkgs/free
  - https://mirrors.tuna.tsinghua.edu.cn/anaconda/cloud/pytorch
  - anaconda
  - https://mirrors.tuna.tsinghua.edu.cn/anaconda/pkgs/main
  - https://mirrors.tuna.tsinghua.edu.cn/anaconda/cloud/pytorch/
  - https://mirrors.tuna.tsinghua.edu.cn/anaconda/cloud/msys2/
  - https://mirrors.tuna.tsinghua.edu.cn/anaconda/cloud/conda-forge
  - https://mirrors.tuna.tsinghua.edu.cn/anaconda/pkgs/free/
  - defaults
dependencies:
  - _libgcc_mutex=0.1=main
  - blas=1.0=mkl
  - ca-certificates=2019.11.27=0
  - certifi=2019.11.28=py36_0
  - cffi=1.13.2=py36h2e261b9_0
  - cuda100=1.0=0
  - cudatoolkit=9.0=h13b8566_0
  - freetype=2.5.5=2
  - intel-openmp=2019.4=243
  - jbig=2.1=hdba287a_0
  - jpeg=9b=h024ee3a_2
  - libedit=3.1.20181209=hc058e9b_0
  - libffi=3.2.1=hd88cf55_4
  - libgcc-ng=9.1.0=hdf63c60_0
  - libgfortran-ng=7.3.0=hdf63c60_0
  - libpng=1.6.37=hbc83047_0
  - libstdcxx-ng=9.1.0=hdf63c60_0
  - libtiff=4.0.9=he85c1e1_2
  - mkl=2018.0.3=1
  - mkl-service=1.1.2=py36h90e4bf4_5
  - mkl_fft=1.0.6=py36h7dd41cf_0
  - mkl_random=1.0.1=py36h4414c95_1
  - ncurses=6.1=he6710b0_1
  - ninja=1.9.0=py36hfd86e86_0
  - numpy=1.15.4=py36h1d66e8a_0
  - numpy-base=1.15.4=py36h81de0dd_0
  - olefile=0.46=py_0
  - openssl=1.0.2u=h7b6447c_0
  - pip=19.3.1=py36_0
  - pycparser=2.19=py_0
  - python=3.6.3=h6c0c0dc_5
  - pytorch=1.0.0=py3.6_cuda10.0.130_cudnn7.4.1_1
  - readline=7.0=h7b6447c_5
  - setuptools=42.0.2=py36_0
  - six=1.13.0=py36_0
  - sqlite=3.30.1=h7b6447c_0
  - tbb=2019.8=hfd86e86_0
  - tbb4py=2019.8=py36hfd86e86_0
  - tk=8.6.8=hbc83047_0
  - wheel=0.33.6=py36_0
  - xz=5.2.4=h14c3975_4
  - zlib=1.2.11=h7b6447c_3
  - zstd=1.3.3=h84994c4_0
  - pip:
    - channelnorm-cuda==0.0.0
    - chardet==3.0.4
    - correlation-cuda==0.0.0
    - cycler==0.10.0
    - decorator==4.4.1
    - dominate==2.4.0
    - idna==2.8
    - imageio==2.8.0
    - kiwisolver==1.1.0
    - matplotlib==3.1.3
    - networkx==2.4
    - opencv-python==4.1.2.30
    - pillow==6.1.0
    - pyparsing==2.4.6
    - python-dateutil==2.8.1
    - pytz==2019.3
    - pywavelets==1.1.1
    - requests==2.22.0
    - resample2d-cuda==0.0.0
    - scikit-image==0.16.2
    - scipy==1.4.1
    - torchvision==0.2.1
    - tqdm==4.19.9
    - urllib3==1.25.7
prefix: /lustre/home/acct-meqf/meqf-3/.conda/envs/pytorch1.0

**Comment by birdflyto** [comment]
i've solved it,it's the misbatch of the enviornment.you can refer to my enviornment.yaml``
name: pytorch1.0
channels:

https://mirrors.tuna.tsinghua.edu.cn/anaconda/pkgs/free
https://mirrors.tuna.tsinghua.edu.cn/anaconda/cloud/pytorch
anaconda
https://mirrors.tuna.tsinghua.edu.cn/anaconda/pkgs/main
https://mirrors.tuna.tsinghua.edu.cn/anaconda/cloud/pytorch/
https://mirrors.tuna.tsinghua.edu.cn/anaconda/cloud/msys2/
https://mirrors.tuna.tsinghua.edu.cn/anaconda/cloud/conda-forge
https://mirrors.tuna.tsinghua.edu.cn/anaconda/pkgs/free/
defaults
dependencies:
_libgcc_mutex=0.1=main
blas=1.0=mkl
ca-certificates=2019.11.27=0
certifi=2019.11.28=py36_0
cffi=1.13.2=py36h2e261b9_0
cuda100=1.0=0
cudatoolkit=9.0=h13b8566_0
freetype=2.5.5=2
intel-openmp=2019.4=243
jbig=2.1=hdba287a_0
jpeg=9b=h024ee3a_2
libedit=3.1.20181209=hc058e9b_0
libffi=3.2.1=hd88cf55_4
libgcc-ng=9.1.0=hdf63c60_0
libgfortran-ng=7.3.0=hdf63c60_0
libpng=1.6.37=hbc83047_0
libstdcxx-ng=9.1.0=hdf63c60_0
libtiff=4.0.9=he85c1e1_2
mkl=2018.0.3=1
mkl-service=1.1.2=py36h90e4bf4_5
mkl_fft=1.0.6=py36h7dd41cf_0
mkl_random=1.0.1=py36h4414c95_1
ncurses=6.1=he6710b0_1
ninja=1.9.0=py36hfd86e86_0
numpy=1.15.4=py36h1d66e8a_0
numpy-base=1.15.4=py36h81de0dd_0
olefile=0.46=py_0
openssl=1.0.2u=h7b6447c_0
pip=19.3.1=py36_0
pycparser=2.19=py_0
python=3.6.3=h6c0c0dc_5
pytorch=1.0.0=py3.6_cuda10.0.130_cudnn7.4.1_1
readline=7.0=h7b6447c_5
setuptools=42.0.2=py36_0
six=1.13.0=py36_0
sqlite=3.30.1=h7b6447c_0
tbb=2019.8=hfd86e86_0
tbb4py=2019.8=py36hfd86e86_0
tk=8.6.8=hbc83047_0
wheel=0.33.6=py36_0
xz=5.2.4=h14c3975_4
zlib=1.2.11=h7b6447c_3
zstd=1.3.3=h84994c4_0
pip:
channelnorm-cuda==0.0.0
chardet==3.0.4
correlation-cuda==0.0.0
cycler==0.10.0
decorator==4.4.1
dominate==2.4.0
idna==2.8
imageio==2.8.0
kiwisolver==1.1.0
matplotlib==3.1.3
networkx==2.4
opencv-python==4.1.2.30
pillow==6.1.0
pyparsing==2.4.6
python-dateutil==2.8.1
pytz==2019.3
pywavelets==1.1.1
requests==2.22.0
resample2d-cuda==0.0.0
scikit-image==0.16.2
scipy==1.4.1
torchvision==0.2.1
tqdm==4.19.9
urllib3==1.25.7
